crates/re_log_types/src/time.rs

```diff
@@ -63,19 +63,23 @@ impl Time {
     }
 
     /// Human-readable formatting
-    pub fn format(&self) -> String {
+    pub fn format(&self, show_timestamps_in_local_timezone: bool) -> String {
         let nanos_since_epoch = self.nanos_since_epoch();
 
         if let Some(datetime) = self.to_datetime() {
+            let local_offset = UtcOffset::current_local_offset();
+            let will_show_timestamps_in_local_timezone =
+                show_timestamps_in_local_timezone && local_offset.is_ok();
```

Comment:
```suggestion
            let local_offset = show_timestamps_in_local_timezone.then(|| UtcOffset::current_local_offset());
```

Reply:
Thanks! With `let local_offset = (time_zone_for_timestamps == TimeZone::Local).then(|| UtcOffset::current_local_offset());` I get:
```
error: redundant closure
   --> crates/re_log_types/src/time.rs:154:87
    |
154 |                 let local_offset = (time_zone_for_timestamps == TimeZone::Local).then(|| UtcOffset::current_local_offset());
    |                                                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: replace the closure with the function itself: `UtcOffset::current_local_offset`
```
Changing to `let local_offset = (time_zone_for_timestamps == TimeZone::Local).then(UtcOffset::current_local_offset);` fixes this. Any concerns?